assets/queries/ansible/aws/user_data_contains_rsa_private_key/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  user_data := task["community.aws.ec2_lc"].user_data
+  contains(user_data, "LS0tLS1CR")	
```

Comment:
Shouldn't you be decoding the data first, form Base64, and then checking if the decoded key contain that substring?

Reply:
No, since certificates are used for authentication, "LS0tLS1CR" is the prefix of any base64 encoded certificate.
Therefore, it is not necessary.

Check here: https://www.base64encode.org/